tests/queries/0_stateless/03604_test_merge_tree_min_read_task_size_is_zero.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+TABLE_NAME="table_${RANDOM}${RANDOM}"
+
+$CLICKHOUSE_CLIENT -q "SET allow_experimental_lightweight_update = 1"
+
+# catch error BAD_ARGUMENTS
+$CLICKHOUSE_CLIENT -q "SET merge_tree_min_read_task_size = 0; -- { serverError BAD_ARGUMENTS }"
+
+$CLICKHOUSE_CLIENT -q "CREATE TABLE ${TABLE_NAME} (c0 Int) ENGINE = MergeTree() ORDER BY tuple()
+ SETTINGS enable_block_number_column = 1, enable_block_offset_column = 1;"
```

Comment:
Let's also add another case where we first create the table and then try to alter the table with the invalid setting.

Reply:
yes, you're right. I didn't notice that it's a session level setting earlier.